0011-interledger-payment-request/0011-interledger-payment-request.md

```diff
@@ -99,42 +62,17 @@ Additional binary headers, encoded as a base64 string, that should be included i
 
 It is RECOMMENDED that recipients use a Message Authentication Code (MAC) of the details of their payment request in the condition so that the condition can be used as an integrity check on incoming transfers. This allows a recipient to ensure that a transfer matches a request they previously generated, without the recipient needing to maintain a log of all outstanding payment requests.
```

Comment:
It's not so much that you're adding a MAC, it's that you're using the condition (which you need to pick anyway) *as* a MAC. So I think something like "you should use the condition as a mac by making the fulfillment a hash of the packet" would be easier to understand.